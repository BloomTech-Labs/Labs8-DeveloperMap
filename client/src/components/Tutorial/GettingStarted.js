import React from 'react';
import signin from '../../images/newsignin.PNG';
import sorc from '../../images/sorc.PNG';

import { StartContainer, Next } from './TutorialStyle';

class GettingStarted extends React.Component {
  switchPage = () => {
    this.props.history.push('/tutorial/navigation');
  };
  render() {
    return (
      <StartContainer>
        <h1>Getting Started</h1>
        <p>
          You can start by signing up through the Sign In button in the top
          right corner of the App. This is also where you will sign in once your
          account is set up.
        </p>
        <img src={signin} alt="" style={{ width: '200px' }} />
        <p>
          Click on the 'Register Here' link to start filling out your
          information needed to use the application effectively. You will be
          prompted to choose whether you are a Job Seeker or an Employer.
        </p>
        <img src={sorc} alt="" style={{ width: '200px' }} />
        <p>
          Next Enter your Email and Password that you would like to affiliate
          with your account. Note: You will be re-entering these in just a
          second. Now just fill out the Sign up form with your info and submit.
          You will then be redireced to the Map.
        </p>
        <Next onClick={this.switchPage}>Next</Next>
      </StartContainer>
    );
  }
}

export default GettingStarted;
