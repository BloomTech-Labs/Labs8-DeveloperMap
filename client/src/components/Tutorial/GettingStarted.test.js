import React from 'react';
import renderer from 'react-test-renderer';
import GettingStarted from './GettingStarted';
import { shallow } from 'enzyme';

describe(`getting started Component`, () => {
    it('renders correctly', () => {
        const tree = shallow(
          <GettingStarted />
        );
        expect(tree).toMatchSnapshot();
      });

});