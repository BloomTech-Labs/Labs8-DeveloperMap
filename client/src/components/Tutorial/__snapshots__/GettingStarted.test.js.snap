// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getting started Component renders correctly 1`] = `
<Styled(styled.div)>
  <h1>
    Getting Started
  </h1>
  <p>
    You can start by signing up through the Sign In button in the top right corner of the App. This is also where you will sign in once your account is set up.
  </p>
  <img
    alt=""
    src="newsignin.PNG"
    style={
      Object {
        "width": "200px",
      }
    }
  />
  <p>
    Click on the 'Register Here' link to start filling out your information needed to use the application effectively. You will be prompted to choose whether you are a Job Seeker or an Employer.
  </p>
  <img
    alt=""
    src="sorc.PNG"
    style={
      Object {
        "width": "200px",
      }
    }
  />
  <p>
    Next Enter your Email and Password that you would like to affiliate with your account. Note: You will be re-entering these in just a second. Now just fill out the Sign up form with your info and submit. You will then be redireced to the Map.
  </p>
  <styled.button
    onClick={[Function]}
  >
    Next
  </styled.button>
</Styled(styled.div)>
`;
